pkg/spoke/spokeagent.go

```diff
@@ -260,36 +261,38 @@ func (o *SpokeAgentOptions) RunSpokeAgent(ctx context.Context, controllerContext
 	if err != nil {
 		return err
 	}
-	spokeClusterInformers := clusterv1informers.NewSharedInformerFactory(spokeClusterClient, 10*time.Minute)
-
-	// create managedClusterClaimController to sync cluster claims
-	managedClusterClaimController := managedcluster.NewManagedClusterClaimController(
-		o.ClusterName,
-		o.MaxCustomClusterClaims,
-		hubClusterClient,
-		hubClusterInformerFactory.Cluster().V1().ManagedClusters(),
-		spokeClusterInformers.Cluster().V1alpha1().ClusterClaims(),
-		controllerContext.EventRecorder,
-	)
 
 	go hubKubeInformerFactory.Start(ctx.Done())
 	go hubClusterInformerFactory.Start(ctx.Done())
 	go spokeKubeInformerFactory.Start(ctx.Done())
 	go namespacedSpokeKubeInformerFactory.Start(ctx.Done())
-	go spokeClusterInformers.Start(ctx.Done())
```

Comment:
I would keep this with the others.  An unused informer .start is a no-op.

Reply:
done